pkg/scheduler/scheduler.go

```diff
@@ -269,7 +269,7 @@ func (sched *Scheduler) Config() *factory.Config {
 // NOTE: This function modifies "pod". "pod" should be copied before being passed.
 func (sched *Scheduler) recordSchedulingFailure(pod *v1.Pod, err error, reason string, message string) {
 	sched.config.Error(pod, err)
-	sched.config.Recorder.Event(pod, v1.EventTypeWarning, "FailedScheduling", message)
+	sched.config.Recorder.Eventf(pod, nil, v1.EventTypeWarning, "FailedScheduling", "", message)
```

Comment:
@bsalamat - I would like you to at least review the exact events (i.e whether related, action and other fields here are exactly what we want).
[Both here an in case of all other events.]